cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -13,12 +13,28 @@ using namespace facebook;
 namespace gluten {
 
 namespace {
+// Spark configs
 const std::string kSparkBatchSize = "spark.sql.execution.arrow.maxRecordsPerBatch";
+const std::string kSparkOffHeapMemory = "spark.gluten.memory.offHeap.size.in.bytes";
+
+// Velox configs
+const std::string kSpillEnabled = "spark.gluten.sql.columnar.backend.velox.spillEnabled";
+const std::string kAggregationSpillEnabled = "spark.gluten.sql.columnar.backend.velox.aggregationSpillEnabled";
```

Comment:
same as above

Reply:
Let's only move common configs. The others were for Velox backend.

Reply:
Do we need a file named `VeloxConfig` to track velox config?

Reply:
Probably. Let's do that in a separate PR?

Reply:
ok